.semaphore/semaphore.yml

```diff
@@ -27,63 +27,55 @@ blocks:
-          commands:
-            - .ci/validation.sh check-missing-pitests
-
-        - name: Eclipse static analysis
-          priority:
-            - value: 10
-              when: true
-          commands:
-            - .ci/validation.sh eclipse-static-analysis
-
-        - name: Ensure that all Sevntu check are used
-          priority:
-            - value: 10
-              when: true
-          commands:
-            - .ci/validation.sh all-sevntu-checks
-
         - name: NonDex (openjdk8)
           priority:
-            - value: 80
+            - value: 90
               when: true
+          matrix:
+            - env_var: CMD
+              values:
+                - .ci/validation.sh nondex
           commands:
             - sem-version java 8 # NonDex only supports Java 8
-            - .ci/validation.sh nondex
-
-        - name: No error test on pmd
-          priority:
-            - value: 80
-              when: true
-          commands:
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-error-pmd
+            - echo "eval of CMD is starting";
+            - echo "CMD=$CMD";
+            - eval $CMD;
+            - echo "eval of CMD is completed";
 
-        - name: No error test on Configurate
+        - name: Validation (openjdk11, slow pool)
           priority:
             - value: 80
               when: true
+          matrix:
+            - env_var: CMD
+              values:
+                - mvn -e clean install -Pno-validations
+                    && .ci/validation.sh no-error-pmd
+                - mvn -e clean install -Pno-validations
+                    && .ci/validation.sh no-violation-test-configurate
           commands:
-            - sem-version java 14
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-violation-test-configurate
-
-        - name: No violation testing on josm
-          commands:
-            - mvn -e install -Pno-validations
-            - .ci/validation.sh no-violation-test-josm
+            - sem-version java 11
+            - echo "eval of CMD is starting";
+            - echo "CMD=$CMD";
+            - eval $CMD;
+            - echo "eval of CMD is completed";
 
-        - name: No exception test
+        - name: Validation (openjdk11, fast pool)
           matrix:
-            - env_var: PROJECT
+            - env_var: CMD
               values:
-                - guava-with-google-checks
-                - guava-with-sun-checks
+                - mvn -e clean install -Pno-validations
+                    && .ci/no-exception-test.sh guava-with-google-checks
+                - mvn -e clean install -Pno-validations
+                    && .ci/no-exception-test.sh guava-with-sun-checks
+                - .ci/validation.sh all-sevntu-checks
+                - .ci/validation.sh check-missing-pitests
+                - .ci/validation.sh eclipse-static-analysis
+                - mvn -e clean install -Pno-validations
```

Comment:
put fast in front... it will also group them a better  to read way
make it like:
```
                - .ci/validation.sh all-sevntu-checks
                - .ci/validation.sh check-missing-pitests
                - .ci/validation.sh eclipse-static-analysis
                - mvn -e clean install -Pno-validations
                    && .ci/no-exception-test.sh guava-with-google-checks
                - mvn -e clean install -Pno-validations
                    && .ci/no-exception-test.sh guava-with-sun-checks
                - mvn -e clean install -Pno-validations
                    && .ci/validation.sh no-violation-test-josm
```

Reply:
done